src/GHC/Util/FreeVars.hs

```diff
@@ -0,0 +1,259 @@
+{-# LANGUAGE CPP #-}
```

Comment:
Why is CPP on here?

Reply:
Get rid of that - it supports GHC's older than HLint does. Just delete the lines.